Package.resolved

```diff
@@ -42,8 +42,8 @@
         "repositoryURL": "https://github.com/mapbox/mapbox-directions-swift.git",
         "state": {
           "branch": null,
-          "revision": "f95be4f764acb0ea40ab11184e09825d3046d14f",
```

Comment:
Do these dependency changes need to be in this commit?

Reply:
> A couple of other observations:
> 
> * There seems to be a problem with the `assemble` GitHub workflow – you haven't removed the Objective-C examples from `Scripts/assemble.sh` but the workflow is still passing. Looking at the workflow logs, there are definitely things failing inside that script. I think you'll need to investigate why workflow isn't hence failing.
> * I think you can also remove the `@objc` on `MapboxConfiguration.init(mapboxKey:)`.

b40d1fa2472a89845d37775e13630befcd0a4118